CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
+# dbt_jira v0.10.1
+## 🐞 Bug Fix
+- Updated logic for model `int_jira__issue_sprint` to further adjust how current sprint is determined. It now uses a combination of the newest `updated_at` date for the issue and the newest `started_at` date of the sprint. This is to account for times when jira updates two sprint records at the same time. ([#77](https://github.com/fivetran/dbt_jira/pull/77))
```

Comment:
Maybe add #78 to the CHANGELOG bugfix as well.
```suggestion
- Updated logic for model `int_jira__issue_sprint` to further adjust how current sprint is determined. It now uses a combination of the newest `updated_at` date for the issue and the newest `started_at` date of the sprint. This is to account for times when jira updates two sprint records at the same time. ([#77](https://github.com/fivetran/dbt_jira/pull/77) and [#78](https://github.com/fivetran/dbt_jira/pull/78))
```

Reply:
Additionally, while this is a patch update it may also require a full refresh. Would you be able to add a note that users may want to run a `dbt run --full-refresh` after upgrading just to ensure they have the latest incremental logic.